### ISSUE TITLE
Add Download in Paused State - Drag and Drop and Add URL(s) Dialog

### ISSUE THREAD

**Issue by hl2guide** [erickutcher/httpdownloader#71]
Please consider adding an **Add Paused** button to the **Add URL(s)** dialog which adds downloads in a paused state.

![add paused](https://user-images.githubusercontent.com/17044982/62444155-1535ab80-b7a0-11e9-9b22-63ddad6cd29f.png)

An option on the **Advanced** tab could be changed to allow **add** or **download** (see screenshot)

![related option](https://user-images.githubusercontent.com/17044982/62444165-1a92f600-b7a0-11e9-8b62-3b2552237ef8.PNG)

Maybe a drop down select or a boolean control?

**Comment by erickutcher** [comment]
Queuing them would be more appropriate. Paused downloads maintain a connection, but don't actually transfer anything. Queued downloads don't maintain a connection and can't transfer anything. There's some issues implementing it that way. I'll have to think about it.

**Comment by erickutcher** [comment]
I've added an Add button between Download and Cancel. It'll add a download to the main window, but it will be in the Stopped state. This was the most appropriate solution I could come up with for this scenario.

If I added the download in the Paused state, then it would have to initiate a transfer first and it would basically be a wasted connection (it'll either time out or the server will close it). When that happens, any queued download will start. That would be unwanted behavior. If I added the download in the Queued state and there were already transfers going, then when those transfers finish, the queued items will start. More unwanted behavior.

So the solution is to have the downloads added in the Stopped state.

I'll make a pair of radio buttons for the Advanced options. One will let you download immediately, and the other will add it in the Stopped state. I'll also add a command-line switch to do the same.
